ios/FluentUI/Controls/Button.swift

```diff
@@ -121,6 +127,8 @@ open class Button: UIButton {
         }
     }
 
+	private var isUsingCustomContentEdgeInsets: Bool = false
```

Comment:
your padding is off. please make sure you are using spaces instead of tab and also if you haven't install https://github.com/microsoft/fluentui-apple#swift-lint

Reply:
It should all be fixed on my third commit to the branch....